Poker.AT/ASAFF/ASAFF.csproj

```diff
@@ -1,129 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i just noticed it now  - all the AT project is inside of `ASAFF/` folder? why? ❓